<ui version="4.0" >
 <class>QgsNewOGRConnection</class>
 <widget class="QDialog" name="QgsNewOGRConnection" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create a New OGR connection</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>338</x>
     <y>10</y>
     <width>89</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnOk" >
      <property name="text" >
       <string>OK</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
      <property name="default" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel" >
      <property name="text" >
       <string>Cancel</string>
      </property>
      <property name="shortcut" >
       <string/>
      </property>
      <property name="autoDefault" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>87</width>
        <height>121</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="Q3GroupBox" name="GroupBox1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>322</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title" >
    <string>Connection Information</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel1_2" >
          <property name="text" >
           <string>Name</string>
          </property>
          <property name="buddy" >
           <cstring>txtName</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel2" >
          <property name="text" >
           <string>Database</string>
          </property>
          <property name="buddy" >
           <cstring>txtDatabase</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel3" >
          <property name="text" >
           <string>Username</string>
          </property>
          <property name="buddy" >
           <cstring>txtUsername</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel3_2" >
          <property name="text" >
           <string>Password</string>
          </property>
          <property name="buddy" >
           <cstring>txtPassword</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="cmbType" >
          <item>
           <property name="text" >
            <string>Oracle</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>PostgreSQL</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>MySQL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtName" >
          <property name="toolTip" >
           <string>Name of the new connection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtServer" />
        </item>
        <item>
         <widget class="QLineEdit" name="txtDatabase" />
        </item>
        <item>
         <widget class="QLineEdit" name="txtUsername" />
        </item>
        <item>
         <widget class="QLineEdit" name="txtPassword" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chkStorePassword" >
        <property name="text" >
         <string>Save Password</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
